Src/BootCamp.Chapter/Items/Armour.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter.Items
+{
+    public class Armour : Item
+    {
+        public float Defense { get; private set; }
```

Comment:
Can be readonly. No need setter altogether.

Reply:
Fixed.